src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingController.kt

```diff
@@ -40,6 +60,13 @@ class BehandlingController(private val behandlingService: BehandlingService) {
         @PathVariable behandlingId: Long,
         @RequestBody endreBehandlendeEnhet: EndreBehandlendeEnhetDto
     ): ResponseEntity<Ressurs<BehandlingResponsDto>> {
+        tilgangService.validerTilgangTilHandlingOgFagsakForBehandling(
+            behandlingId = behandlingId,
+            minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
+            event = AuditLoggerEvent.ACCESS,
```

Comment:
Skulle det vært UPDATE her?

```suggestion
            event = AuditLoggerEvent.UPDATE,
```

Reply:
Good catch 🤩